bochs/instrument/stubs/instrument.cc

```diff
@@ -24,6 +24,8 @@
 
 #if BX_INSTRUMENTATION
 
+class bxInstruction_c {};
```

Comment:
are you sure empty class is needed or just forward declaration ?
What was you configure script and what was the compilation error ?

Reply:
The configure script is the script in the repo:

```
./configure --enable-instrumentation && make
```

fails with

```
g++ -c -I../.. -I./../.. -I. -I./. -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES -g -O2 -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES -pthread instrument.cc -o instrument.o
instrument.cc:44:36: error: ‘bxInstruction_c’ has not been declared
   44 | void bx_instr_opcode(unsigned cpu, bxInstruction_c *i, const Bit8u *opcode, unsigned len, bool is32, bool is64) {}
      |                                    ^~~~~~~~~~~~~~~
instrument.cc:55:46: error: ‘bxInstruction_c’ has not been declared
   55 | void bx_instr_before_execution(unsigned cpu, bxInstruction_c *i) {}
      |                                              ^~~~~~~~~~~~~~~
instrument.cc:56:45: error: ‘bxInstruction_c’ has not been declared
   56 | void bx_instr_after_execution(unsigned cpu, bxInstruction_c *i) {}
      |                                             ^~~~~~~~~~~~~~~
instrument.cc:57:46: error: ‘bxInstruction_c’ has not been declared
   57 | void bx_instr_repeat_iteration(unsigned cpu, bxInstruction_c *i) {}
      |                                              ^~~~~~~~~~~~~~~
make[1]: *** [Makefile:52: instrument.o] Error 1
```

A declaration is good enough, I’ll amend the PR.